dynamodb/dynamodb.tf-aws.w

```diff
@@ -56,7 +59,11 @@ pub class Table_tfaws impl dynamodb_types.ITable {
       attribute: props.attributes,
       hashKey: props.hashKey,
       rangeKey: props.rangeKey,
-      billingMode: "PAY_PER_REQUEST",
+      billingMode: "{this.billingMode}",
+      deletionProtectionEnabled: props.deletionProtection ?? false,
+      lifecycle: {
+        preventDestroy: props.deletionProtection ?? true
```

Comment:
Should these fields default to the same value?

Reply:
Oh that's a good catch. Yes that would make sense. I actually was wondering if there has been a wider discussion about having a system wide sensible default for those things.

Reply:
I don't think there's any built-in mechanisms at the moment -- perhaps we can add `deletionProtection` as a prop for more resources and eventually establish it as a convention of some sort.